model.py

```diff
@@ -9380,7 +9380,7 @@ def remove_entry(self, work_or_edition):
         if isinstance(work_or_edition, Work):
```

Comment:
I don't think you need these three lines anymore since this method no longer uses `edition`.